### PR TITLE
Add example of instance_type per role

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

File: appscale/tools/templates/AppScalefile [hunk 1]
```diff
@@ -23,19 +23,27 @@ ips_layout:
       - zookeeper
     nodes: [1 | vm1_private_IP]
 
-# A multi node deployment.
+# A multi node deployment example. Within nodes one can use the numbers of
```
Comment (tmarballi) [inline appscale/tools/templates/AppScalefile]: numbers -> number 
Maybe simplify it to "Within nodes, one can specify either the number of nodes desired (cloud deployment) or a list of IPs (cluster deployment)."

File: appscale/tools/templates/AppScalefile [hunk 2]
```diff
@@ -75,8 +83,11 @@ ips_layout:
 #EC2_ACCESS_KEY: 'YYYY'
 #EC2_SECRET_KEY: 'ZZZZ'
 
-# Instance type of your choice. Default is m3.medium.
-#instance_type : 'm3.medium'
+# This is the default instance type to be used when no instance_type has
+# been specified for the particular node. It is also the instance type
+# that will be used for autoascaling (if autoscaling is enabled). It can
```
Comment (whoarethebritons) [inline appscale/tools/templates/AppScalefile]: autoascaling

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/5884/

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/5884/

**Pull request merged**